CHANGES.rst

```diff
@@ -297,6 +297,10 @@ Other Changes and Additions
   manage the version number, and adding a ``pyproject.toml`` to opt in to
   isolated builds as described in PEP 517/518. [#9726]
 
+- Add a known issue that using ``float`` on standard complex numbers
```

Comment:
no need for changelog entry for docs changes

Reply:
Removed, assuming I know how to revert things with git.

Reply:
Awesome!  I do know how to revert things with git!  :tada: :birthday: :tada: :birthday: :balloon:   The small victories in life...

Reply:
is it too bad if I ask you to squash them? We don't really need to keep reverts in the history. (on the other hand I'm happy to squash everything before merging, once the CI is happy with it).